modules/apps/object/object-api/src/main/java/com/liferay/object/service/ObjectDefinitionLocalServiceWrapper.java

```diff
@@ -629,14 +629,6 @@ public boolean hasObjectRelationship(long objectDefinitionId) {
 			userId, objectDefinitionId);
 	}
 
-	@Override
-	public void unbindObjectDefinition(long objectDefinitionId)
-		throws com.liferay.portal.kernel.exception.PortalException {
-
-		_objectDefinitionLocalService.unbindObjectDefinition(
-			objectDefinitionId);
-	}
```

Comment:
Hey @victorKammerer please move this change to the Build Service commit.

Thank you!

Reply:
Done, thanks!